### ISSUE TITLE
Launcher: org.osgi.framework.startlevel.beginning property at runtime is always "1"

### ISSUE THREAD

**Issue by glimmerveen** [bndtools/bnd#5693]
When in a running application I try to introspect the `org.osgi.framework.startlevel.beginning` property, I found that it always returns "1", even though in the launcher.properties this property may be assigned another value.

Is the fixed assigning of "1" by the StartLevelRuntimeHandler a bug, or is there a reason for this?

https://github.com/bndtools/bnd/blob/25e4898599326264c57da8b695ae5df9e268891e/aQute.libg/src/aQute/lib/startlevel/StartLevelRuntimeHandler.java#L167

**Comment by pkriens** [comment]
Yes. I am a bit fuzzy on the details but it had to do that we needed to do something _after_ the framework was started but _before_ the bundles were started. So we set it at 1, this will start the framework but no bundles. The launcher then configures the bundles' startlevels, and then sets the actual framework start level which will start the bundles.

Please close this if this is a sufficient answer.
